### DISCUSSION TITLE
Waiting for the benefits although approved

### DISCUSSION THREAD

**Discussion by HvTung04** [community/community#62746] [GitHub Education] [answered]
### Hi everyone,

Hi everyone, my request have been approved, since then I have never recieved any further information. May I ask when would I recieve any more information. Thank you!
![image](https://github.com/community/community/assets/91996589/52c8454f-85ba-4980-ad18-b499a9009d26)

**Comment by prabhjotschugh** [comment, marked as answer]
Hi
After approval, it can take about 5-7 days to receive the benefits. You will get an email regarding the same.
Hope it helps!

**Reply by HvTung04** [reply]
Thank you for your help sir!

**Reply by prabhjotschugh** [reply]
kindly mark it as answered, for others to know.
Thank you
